tests/TestCase/Database/Driver/SqliteTest.php

```diff
@@ -143,4 +144,37 @@ public function testSchemaValue($input, $expected)
         $driver->connection($mock);
         $this->assertEquals($expected, $driver->schemaValue($input));
     }
+    
+    /**
+     * Tests that GROUP_CONCAT is transformed correctly
+     *
+     * @return void
+     */
+    public function testGroupConcatTransform()
+    {
+        $driver = $this->getMockBuilder('Cake\Database\Driver\Sqlite')
```

Comment:
The rest of the file is using string syntax also, might be something for a more general cleanup.,,